src/plugins/kernel_install/utils.sh

```diff
@@ -550,6 +549,18 @@ function do_uninstall()
   fi
 }
 
+function is_in_array()
+{
+  target_element=$1
+  array=("${!2}")
```

Comment:
Please, declare the local array together with target_element and add one new line after it.